doc/api-extensions.md

```diff
@@ -2457,3 +2457,8 @@ This can be used by a virtual machine running LXD to access raw images from the
 
 Adds a new `virtio-blk` value for `io.bus` on `disk` devices which allows
 for the attached disk to be connected to the `virtio-blk` bus.
+
+## `ubuntu_pro_guest_attach`
```

Comment:
Please can the commit message start with `api: ` as per the documented convention:

https://github.com/canonical/lxd/blob/main/CONTRIBUTING.md#commit-structure